src/js/base/fontawesome.js

```diff
@@ -6,9 +6,9 @@ import '@fortawesome/fontawesome-svg-core/styles.css';
 config.autoAddCss = false;
 config.replacementClass = 'svg-inline--fa icon';
 
-function faHelper(prefix, iconName, { hash = {} }) {
+function faHelper(prefix, iconName, params = {}) {
```

Comment:
I'm not sure why this was written as `{ hash = {} }` before... seems like a typo?

Reply:
This is a handlebars thing that is probably worth keeping. https://handlebarsjs.com/guide/block-helpers.html#hash-arguments
Also this change is definitely breaking a `fa*` template helper somewhere, but it isn't going to be caught by tests as it's a style change.

Reply:
I don't follow. I'm pretty sure that the handlebar docs are saying that the final argument for a helper should be a hash (aka plain object) with a default of `{}`, like I am showing here.

The `{ hash = {} }` makes a required object with an optional `hash` property... right?

Reply:
For instance for the handlebars block `if` `{{#if foo includeZero=true}}`  is checked on this line: https://github.com/handlebars-lang/handlebars.js/blob/6790c080c641ef2b44e663800e1794fae180977a/lib/handlebars/helpers/if.js#L16